.chloggen/node-label-opt.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
I wonder if this should be `breaking`? @open-telemetry/specs-semconv-approvers thoughts?

Reply:
I think it would be considered breaking, since instrumentations will no longer capture something by default when they take in this change. at least I think that's how we would treat it if it were a span attribute, not sure if things are different for resource attributes